SwiftUIDZ/ContentView.swift

```diff
@@ -7,16 +7,84 @@
 
 import SwiftUI
 
+// Экран с alert и actionSheet
 struct ContentView: View {
+    private enum Constants {
+        static let oldText = "Старый текст"
+        static let newText = "Новый текст"
+        static let updateText = "Обновить текст"
+        static let alertName = "Показать Блок1"
+        static let alertWithLogicName = "Показать Блок2"
+        static let actionSheetText = "Показать Блок3"
+        static let actionSheetWithLogicText = "Показать Блок4"
+        static let alertTitleText = "Пример Alert"
+        static let alertWithLogicTitleText = "Пример Alert c 2 кнопками и логикой"
+        static let actionSheetTitleText = "Пример ActionSheet"
+        static let actionSheetWithLogicTitleText = "Пример ActionSheet c кнопками и логикой"
+    }
+    
+    @State private var isAlert = false
+    @State private var isAlertWithLogic = false
+    @State private var isActionSheet = false
+    @State private var isActionSheetWithLogic = false
```

Comment:
```suggestion
    @State private var isActionSheetWithLogic = false
```
Все приватные ниже var body